Cirrious.FluentLayout/AdvancedFluentLayoutExtensions.cs

```diff
@@ -56,14 +59,14 @@ public static FluentLayout WithSameTop(this UIView view, UIView previous)
             return view.Top().EqualTo().TopOf(previous);
         }
 
-        public static FluentLayout WithSameCenterX(this UIView view, UIView previous)
+        public static FluentLayout WithSameCenterX(this UIView view, UIView previous, nfloat? margin = null)
         {
```

Comment:
I've not used this library recently enough - so I'm going to just ask this openly... 

Is adding this optional parameter the right thing to do? It doesn't feel very "fluent" to me... would adding a new method with some `WithCenterXPlus` (or something) be more "fluent"?